lib/fog/aws/requests/dns/change_resource_record_sets.rb

```diff
@@ -141,6 +141,7 @@ def change_resource_record_sets(zone_id, change_batch, options = {})
           body = %Q{<?xml version="1.0" encoding="UTF-8"?><ChangeResourceRecordSetsRequest xmlns="https://route53.amazonaws.com/doc/#{@version}/">#{changes}</ChangeResourceRecordSetsRequest>}
           request({
             :body       => body,
+            :idempotent => true,
```

Comment:
I wondered the same thing. Usually I would say no (post generally creates things). So far as I can tell this is more like a put (updating the record) and it just replaces values. So I think you could re-replace with the same values as often or as many times as you might want without negative impact. Definitely let me know if I misunderstood though.